pkl-config-kotlin/api/pkl-config-kotlin.api

```diff
@@ -0,0 +1,24 @@
+public final class org/pkl/config/kotlin/ConfigExtensionsKt {
+	public static final fun forKotlin (Lorg/pkl/config/java/ConfigEvaluator;)Lorg/pkl/config/java/ConfigEvaluator;
+	public static final fun forKotlin (Lorg/pkl/config/java/ConfigEvaluatorBuilder;)Lorg/pkl/config/java/ConfigEvaluatorBuilder;
+	public static final fun forKotlin (Lorg/pkl/config/java/mapper/ValueMapperBuilder;)Lorg/pkl/config/java/mapper/ValueMapperBuilder;
+}
+
```

Comment:
These API pins will ensure that applicable modules (published publicly) have a known interface. Breakages can be checked before merging PRs, to ensure downstream library consumers aren't broken by a given change.

The plugin that produces this is made by JetBrains (available [here](https://github.com/Kotlin/binary-compatibility-validator)). It is used for core libraries like KotlinX Serialization, etc, so it's very reliable and well maintained.

Update pins: `./gradlew apiDump`
Check pins: `./gradlew apiCheck`